### ISSUE TITLE
Unhandled exception. System.NullReferenceException: Object reference not set to an instance of an object.

### ISSUE THREAD

**Issue by mhbosch** [microsoft/AttackSurfaceAnalyzer#561]
**Describe the bug**
Error by Console Command Monitor

**To Reproduce**
Command: Asa.exe monitor -f --verbose

**System Configuration (please complete the following information):**
 - OS: Windows 10

Version: AttackSurfaceAnalyzer v.2.2.74+efd3bdfa22

```
[22:22:08 VRB] Failed to get owner for C:\Users\posts\Downloads\ASA_win_2.2.74\asa.sqlite-journal (System.IO.FileNotFoundException:C:\Users\posts\Downloads\ASA_win_2.2.74\asa.sqlite-journal)
Unhandled exception. Unhandled exception. Unhandled exception. System.NullReferenceException: Object reference not set to an instance of an object.
   at Microsoft.Data.Sqlite.SqliteConnection.RemoveCommand(SqliteCommand command)
   at AttackSurfaceAnalyzer.Utils.DatabaseManager.WriteFileMonitor(FileMonitorObject fmo, String RunId) in D:\a\1\s\Lib\Utils\DatabaseManager.cs:line 979
   at AttackSurfaceAnalyzer.Collectors.FileSystemMonitor.WriteChange(FileSystemEventArgs objIn, NotifyFilters filters) in D:\a\1\s\Lib\Collectors\FileSystemMonitor.cs:line 224
   at System.IO.FileSystemWatcher.NotifyFileSystemEventArgs(WatcherChangeTypes changeType, ReadOnlySpan`1 name)
   at System.IO.FileSystemWatcher.ParseEventBufferAndNotifyForEach(Byte[] buffer, UInt32 numBytes)
   at System.IO.FileSystemWatcher.ReadDirectoryChangesCallback(UInt32 errorCode, UInt32 numBytes, NativeOverlapped* overlappedPointer)
   at System.Threading._IOCompletionCallback.PerformIOCompletionCallback(UInt32 errorCode, UInt32 numBytes, NativeOverlapped* pNativeOverlapped)
System.NullReferenceException: Object reference not set to an instance of an object.
   at Microsoft.Data.Sqlite.SqliteConnection.RemoveCommand(SqliteCommand command)
   at AttackSurfaceAnalyzer.Utils.DatabaseManager.WriteFileMonitor(FileMonitorObject fmo, String RunId) in D:\a\1\s\Lib\Utils\DatabaseManager.cs:line 979
   at AttackSurfaceAnalyzer.Collectors.FileSystemMonitor.WriteChange(FileSystemEventArgs objIn, NotifyFilters filters) in D:\a\1\s\Lib\Collectors\FileSystemMonitor.cs:line 224
   at System.IO.FileSystemWatcher.NotifyFileSystemEventArgs(WatcherChangeTypes changeType, ReadOnlySpan`1 name)
   at System.IO.FileSystemWatcher.ParseEventBufferAndNotifyForEach(Byte[] buffer, UInt32 numBytes)
   at System.IO.FileSystemWatcher.ReadDirectoryChangesCallback(UInt32 errorCode, UInt32 numBytes, NativeOverlapped* overlappedPointer)
   at System.Threading._IOCompletionCallback.PerformIOCompletionCallback(UInt32 errorCode, UInt32 numBytes, NativeOverlapped* pNativeOverlapped)
System.NullReferenceException: Object reference not set to an instance of an object.
   at Microsoft.Data.Sqlite.SqliteConnection.RemoveCommand(SqliteCommand command)
   at AttackSurfaceAnalyzer.Utils.DatabaseManager.WriteFileMonitor(FileMonitorObject fmo, String RunId) in D:\a\1\s\Lib\Utils\DatabaseManager.cs:line 979
   at AttackSurfaceAnalyzer.Collectors.FileSystemMonitor.WriteChange(FileSystemEventArgs objIn, NotifyFilters filters) in D:\a\1\s\Lib\Collectors\FileSystemMonitor.cs:line 224
   at System.IO.FileSystemWatcher.NotifyFileSystemEventArgs(WatcherChangeTypes changeType, ReadOnlySpan`1 name)
   at System.IO.FileSystemWatcher.ParseEventBufferAndNotifyForEach(Byte[] buffer, UInt32 numBytes)
   at System.IO.FileSystemWatcher.ReadDirectoryChangesCallback(UInt32 errorCode, UInt32 numBytes, NativeOverlapped* overlappedPointer)
   at System.Threading._IOCompletionCallback.PerformIOCompletionCallback(UInt32 errorCode, UInt32 numBytes, NativeOverlapped* pNativeOverlapped)
Unhandled exception. System.NullReferenceException: Object reference not set to an instance of an object.
   at Microsoft.Data.Sqlite.SqliteConnection.RemoveCommand(SqliteCommand command)
   at AttackSurfaceAnalyzer.Utils.DatabaseManager.WriteFileMonitor(FileMonitorObject fmo, String RunId) in D:\a\1\s\Lib\Utils\DatabaseManager.cs:line 979
   at AttackSurfaceAnalyzer.Collectors.FileSystemMonitor.WriteChange(FileSystemEventArgs objIn, NotifyFilters filters) in D:\a\1\s\Lib\Collectors\FileSystemMonitor.cs:line 224
   at AttackSurfaceAnalyzer.Collectors.FileSystemMonitor.WriteSecurityChange(Object source, FileSystemEventArgs e) in D:\a\1\s\Lib\Collectors\FileSystemMonitor.cs:line 178
   at System.IO.FileSystemWatcher.NotifyFileSystemEventArgs(WatcherChangeTypes changeType, ReadOnlySpan`1 name)
   at System.IO.FileSystemWatcher.ParseEventBufferAndNotifyForEach(Byte[] buffer, UInt32 numBytes)
   at System.IO.FileSystemWatcher.ReadDirectoryChangesCallback(UInt32 errorCode, UInt32 numBytes, NativeOverlapped* overlappedPointer)
   at System.Threading._IOCompletionCallback.PerformIOCompletionCallback(UInt32 errorCode, UInt32 numBytes, NativeOverlapped* pNativeOverlapped)
Unhandled exception. System.NullReferenceException: Object reference not set to an instance of an object.
   at Microsoft.Data.Sqlite.SqliteConnection.RemoveCommand(SqliteCommand command)
   at AttackSurfaceAnalyzer.Utils.DatabaseManager.WriteFileMonitor(FileMonitorObject fmo, String RunId) in D:\a\1\s\Lib\Utils\DatabaseManager.cs:line 979
   at AttackSurfaceAnalyzer.Collectors.FileSystemMonitor.WriteChange(FileSystemEventArgs objIn, NotifyFilters filters) in D:\a\1\s\Lib\Collectors\FileSystemMonitor.cs:line 224
   at AttackSurfaceAnalyzer.Collectors.FileSystemMonitor.WriteDirectoryNameChange(Object source, FileSystemEventArgs e) in D:\a\1\s\Lib\Collectors\FileSystemMonitor.cs:line 174
   at System.IO.FileSystemWatcher.NotifyFileSystemEventArgs(WatcherChangeTypes changeType, ReadOnlySpan`1 name)
   at System.IO.FileSystemWatcher.ParseEventBufferAndNotifyForEach(Byte[] buffer, UInt32 numBytes)
   at System.IO.FileSystemWatcher.ReadDirectoryChangesCallback(UInt32 errorCode, UInt32 numBytes, NativeOverlapped* overlappedPointer)
   at System.Threading._IOCompletionCallback.PerformIOCompletionCallback(UInt32 errorCode, UInt32 numBytes, NativeOverlapped* pNativeOverlapped)
```

**Comment by gfs** [comment]
Thanks for your report. I'll investigate this week.

**Comment by gfs** [comment]
I've identified the issue and plan to have a fix released today with PR #563

**Comment by gfs** [comment]
Fixed in #563.
